src/form/textarea/Textarea.tsx

```diff
@@ -0,0 +1,137 @@
+import "./_textarea.scss";
+
+import React, {useState} from "react";
+import classNames from "classnames";
+import TextareaAutosize, {TextareaAutosizeProps} from "react-textarea-autosize";
+
+import {KEYBOARD_EVENT_KEY} from "../../core/utils/keyboardEventConstants";
+
+type TextareaProps = Omit<
+  React.TextareaHTMLAttributes<HTMLTextAreaElement>,
+  "name" | "onChange" | "className"
+> & {
+  name: string;
+  onChange: React.ReactEventHandler<HTMLTextAreaElement>;
+  value?: string;
+  testid?: string;
+  isDisabled?: boolean;
+  customClassNames?: {
+    container?: string;
+    textarea?: string;
+  };
+  style?: React.CSSProperties;
+  autoSizeProps?: TextareaAutosizeProps;
+  onJustEnterPressed?: () => void;
+  onShiftEnter?: () => void;
+  isRequired?: boolean;
+  hasError?: boolean;
+};
+
+function Textarea(props: TextareaProps) {
+  const {
+    name,
+    value,
+    isDisabled,
+    customClassNames,
+    testid,
+    onKeyUp,
+    style,
+    autoSizeProps,
+    onJustEnterPressed,
+    isRequired,
+    onShiftEnter,
+    hasError,
+    autoComplete = "off",
+    autoCorrect = "off",
+    autoFocus = false,
+    ...otherProps
+  } = props;
+  const [isShiftPressed, setShiftPressedState] = useState(false);
+  const containerClassName = classNames(
+    "textarea-container",
+    customClassNames?.container,
+    {"textarea--has-error": hasError,
+     "textarea--is-disabled" : isDisabled
+    }
```

Comment:
Is this formatting valid? 🤔  I think it should be like this (according to our linter rules):
```suggestion
    {
     "textarea--has-error": hasError,
     "textarea--is-disabled" : isDisabled
    }
```

Reply:
You're right. I guess we should update our `eslint & stylelint` packages. Can we @edizcelik ?

Reply:
No this should be handled by Prettier